### ISSUE TITLE
Suggest being able to suppress the board rotation after each move

### ISSUE THREAD

**Issue by stsb11** [cjsaylor/chessbot#63]
Love this... Absolutely love it. Best Slack plugin ever.

That said, I find it very difficult to plan my next move when the board is rotated 180 degrees and re-drawn after each move, though. If there was a way to prevent the board from rotating after each move so that it looks the same all the way through the game, this'd be fab.

**Comment by cjsaylor** [comment]
Agreed, I'm thinking of perhaps tapping into some of Slack's newer UI elements to be able to configure the game in various ways. Specifically, the "surface modal": https://api.slack.com/surfaces/modals.

**Comment by stsb11** [comment]
Very kind of you to reply - thank you. Surface Modals sound fab. I'm afraid that I'm just a thicko end user; I'd struggle to implement FizzBuzz in Scratch. Writing anything meaningful that would help your project is beyond my skills, otherwise I'd love to muck in and help.

**Comment by cjsaylor** [comment]
@stsb11 Ideas are just as important. Knowing what players want/need is critical.

**Comment by unmerkbar** [comment]
+1 from me :)
